.changeset/six-radios-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': patch
+---
+
+Draggable dialog now have a min height set so that the dialog can't be minimized more than where the dialog buttons are. It can also be optionally customized using the `$min-height` variable
```

Comment:
This changeset looks unintentional. I think it might be leftover from your old branch.